### PR TITLE
who wouldn't want a big ulimit?

### PR DESCRIPTION
**Pull request by johnduffell**
this time with the branch in the same repo

All as per https://github.com/guardian/amigo/pull/89

@rtyley @LATaylor-guardian

### CODE REVIEW

**Comment by rtyley** [comment]
:+1: using a branch rather than a fork means that the riffraff credentials are now available, right!

**Comment by rtyley** [comment]
@johnduffell looks like amigo is needs to be manually deployed https://riffraff.gutools.co.uk/deployment/history?projectName=amigo&page=1#

File: roles/ubuntu-init/tasks/main.yml [hunk 1]
```diff
@@ -4,3 +4,10 @@
 
 - name: Configure locale
   command: locale-gen en_GB.UTF-8
+
+- name: Bump ulimits
+  blockinfile:
+    dest: /etc/security/limits.conf
+    block: |
+      *  soft  nofile  16384
```
Comment (paulbrown1982) [inline roles/ubuntu-init/tasks/main.yml]: @johnduffell I think this needs to be root and not * for the way we load the apps. the 'frontend' user, a shadow of root I think, has only a limit of 4096 

http://stackoverflow.com/questions/21515463/how-to-increase-maximum-file-open-limit-ulimit-in-ubuntu

Comment (johnduffell) [inline roles/ubuntu-init/tasks/main.yml]: Oh dear didn't realise that.

Comment (rtyley) [inline roles/ubuntu-init/tasks/main.yml]: Did this lead to https://github.com/guardian/amigo/pull/94 to get the limit set for the frontend user?

Comment (johnduffell) [inline roles/ubuntu-init/tasks/main.yml]: in the end it was nothing to do with user limits, actually systemd has service limits that are completely separate, but that's the right PR

**Pull request merged**